presto-main/src/main/java/com/facebook/presto/operator/aggregation/AbstractMinMaxAggregationFunction.java

```diff
@@ -256,4 +269,112 @@ public static void input(MethodHandle methodHandle, BlockState state, Block valu
             throw new PrestoException(GENERIC_INTERNAL_ERROR, t);
         }
     }
+
+    public static void combine(MethodHandle methodHandle, NullableLongState state, NullableLongState otherState)
+    {
+        if (state.isNull() && !otherState.isNull()) {
```

Comment:
I would remove tests for `otherState.isNull` for the same reason I mentioned before.